codecov.yml

```diff
@@ -28,5 +28,8 @@ coverage:
 comment: false
 
 # https://docs.codecov.io/docs/flags
```

Comment:
Unfortunately, this functionality appears to be limited in the full UI currently due to performance issues https://docs.codecov.io/docs/flags#section-flags-in-the-codecov-ui